### DISCUSSION TITLE
How to use  Liquidsoap's docker image

### DISCUSSION THREAD

**Discussion by jonahzheng** [savonet/liquidsoap#1753] [Installation] [answered]
os: ubuntu 18.04
wanted to install version 1.4.3, but could not find the DEB of Ubuntu 18.04.

**Comment by toots** [comment]
Hi! Ubuntu `18.04` has outdated libraries for us so we cannot build a fully featured image with it. What exactly is your issue with the docker image?

**Comment by jonathanstowe** [comment, marked as answer]
I run the docker image like:

```
docker run --entrypoint bash -it -p 9985:9985 -v `pwd`:/home/liquidsoap -w /home/liquidsoap --rm savonet/liquidsoap:v1.4.4 -c "liquidsoap test-resources.liq"
```
Obviously you'd need to adjust the port mapping as per your needs, and the name of the script to be run.
